'''
=> Pyhton exception handling has 4 keywords:
1. try
2. except
3. else 
4. finally 

The try block lets you test a block of code for errors.

The except block lets you handle the error.

The finally block lets you execute code, regardless of the result of the try- and except blocks.


'''